scaffolder-templates/gitops/bootstrap/README.md

```diff
@@ -7,11 +7,14 @@ Before applying the manifests, there is a need to replace the `__REPLACE_SSH_PRI
 Either manually edit the file or login to the target cluster and run the following command:
 
 ```
+# Change the Git host to the one you are using, for example, github.com, gitlab.cee.redhat.com, gitlab.gitlab, etc.
```

Comment:
This comment will likely go unnoticed until there is an error

Could you parametrized the URL here, like using `parameters.gitlabHost`?
Maybe you can have an `if` statement to only do that when GL is used